openpype/hosts/resolve/api/plugin.py

```diff
@@ -349,10 +355,14 @@ def _populate_data(self):
         # create name
         repr = self.context["representation"]
         repr_cntx = repr["context"]
-        asset = str(repr_cntx["asset"])
-        subset = str(repr_cntx["subset"])
-        representation = str(repr_cntx["representation"])
-        self.data["clip_name"] = "_".join([asset, subset, representation])
+        self.asset = str(repr_cntx["asset"])
+        self.subset = str(repr_cntx["subset"])
+        self.representation = str(repr_cntx["representation"])
+        self.data["clip_name"] = "_".join([
+            self.asset,
+            self.subset,
+            self.representation
+        ])
```

Comment:
Is there a reason these need to be stored on `self`? It's making it look like it's used more elsewhere but it's hard to pinpoint exactly where.

Reply:
yeah, this is quite dirty ;)